src/main.ts

```diff
@@ -0,0 +1,215 @@
+import { DEFAULT_SETTINGS } from 'defaults'
+import { Box, CalendarData, CalendarSettings, Entry } from 'model'
+import { Plugin } from 'obsidian'
+import HeatmapCalendarSettingsTab from 'settings'
+
+declare global {
```

Comment:
adding the global declare removes the need for ts-ignore when accessing the window object